packages/core/strapi/lib/services/entity-validator/validators.js

```diff
@@ -131,7 +141,7 @@ module.exports = {
   password: stringValidator,
   email: emailValidator,
   enumeration: enumerationValidator,
-  boolean: () => () => yup.mixed(),
+  boolean: () => yup.boolean(),
```

Comment:
We will need to be careful to allow null

Reply:
This reverted the change to what it was originally. If we accepted `null` before, it should still work.